frontend/packages/console-app/src/components/nav/FavoriteNavItemResources.scss

```diff
@@ -0,0 +1,38 @@
+.oc-favorite-resource.pf-v6-c-nav__item {
```

Comment:
Throughout the file, please alphabetize rules by selector and declarations by property.